pkg/xds/server/diagnostics.go

```diff
@@ -31,7 +31,7 @@ func (s *diagnosticsServer) Start(stop <-chan struct{}) error {
 		resp.WriteHeader(http.StatusOK)
 	})
 
-	httpServer := &http.Server{Addr: fmt.Sprintf(":%d", s.port), Handler: mux}
+	httpServer := &http.Server{Addr: fmt.Sprintf("127.0.0.1:%d", s.port), Handler: mux}
```

Comment:
Please revert this back so the server can be accessed outside of the control plane machine.